src/mito2/src/engine.rs

```diff
@@ -95,6 +95,11 @@ impl MitoEngine {
     pub(crate) fn get_region(&self, id: RegionId) -> Option<crate::region::MitoRegionRef> {
         self.inner.workers.get_region(id)
     }
+
+    /// Returns the estimated size of the region's WAL.
+    pub fn get_region_wal_size(&self, id: RegionId) -> Result<usize> {
```

Comment:
nitpick: put the pub method before `pub(crate)` and private methods (moving to L88).

Reply:
ok